network-api/networkapi/utility/orderables.py

```diff
@@ -1,15 +1,21 @@
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Optional
 
 if TYPE_CHECKING:
     from django.db import models
 
 
-def get_related_items(queryset: "models.QuerySet", related_item_field: str) -> list["models.Model"]:
+def get_related_items(
+    queryset: "models.QuerySet", related_item_field: str, order_by: Optional[str] = None
```

Comment:
We're on Python 3.11 now, I think it's better if we use `str | None` for typing of optional paramaters

Reply:
Updated!